cmake/miaConfig.cmake.in

```diff
@@ -34,24 +36,28 @@ function(miagen target)
 
 	set(mia_output_pattern "${CMAKE_CURRENT_BINARY_DIR}/{}.mia.hpp")
 
+	if (mia_verbose)
```

Comment:
Should it? Maybe we should make it a proper option? You would want to use it for setting up a debug version of a project. I think its better to pass it by cmake option instead of parameter as it better reflects the use case for this option.